buildSrc/build.gradle.kts

```diff
@@ -15,7 +15,7 @@ repositories {
 // Pass '-PkotlinDev' to command line to enable kotlin-in-development version
 val kotlinVersion = if (project.hasProperty("kotlinDev")) {
     logger.warn("Enabling kotlin dev version!")
-    "1.6.0"
+    "1.6.20-M1"
```

Comment:
May you can bump it to 1.6.20-RC2?
https://github.com/JetBrains/kotlin/releases/tag/v1.6.20-RC2